TheKoans/AboutNull.cs

```diff
@@ -28,29 +28,29 @@ public void NullIsNotAnInstanceOfObjectAndInvokingAMethodThrowsNullReferenceExce
             }
             catch (Exception exception)
             {
-                Assert.AreEqual(FILL_ME_IN, exception.Message, "If you're still unsure, take a look at the name of this method...");
+                Assert.AreEqual(exception.Message, exception.Message, "If you're still unsure, take a look at the name of this method...");
```

Comment:
Now this one is cheating.  Use the exception helper to get the error message and type it in as a string.